cli/internal/daemon/connector/connector_test.go

```diff
@@ -0,0 +1,250 @@
+package connector
+
+import (
+	"context"
+	"errors"
+	"net"
+	"os/exec"
+	"runtime"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/nightlyone/lockfile"
+	"github.com/vercel/turborepo/cli/internal/fs"
+	"github.com/vercel/turborepo/cli/internal/turbodprotocol"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/status"
+	"google.golang.org/grpc/test/bufconn"
+	"gotest.tools/v3/assert"
+)
+
+func testBin() string {
```

Comment:
This is clever. Can you comment how we're not actually starting our daemon in this test though?

Reply:
Done.